Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gleam"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
I believe this is a breaking change, and @nox will punish me if I r+ it in this state.